app/views/rock_pebble_mailer/rock_denied_message.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1 style="font-family: 'Oswald'; text-color:#525b86;" width="250">Paired</h1>
+<%= @message %>
+
+<p>Sincerely, <br>The Paired Team</p>
```

Comment:
(and a partial here too)

Reply:
Made partials but just to make sure I understand...we still need a file for each message correct? The partials just clean up the actual files?

Reply:
Yep!